packages/manager/modules/vrack/src/dashboard/vrack.controller.js

```diff
@@ -264,82 +268,366 @@ export default class VrackMoveDialogCtrl {
+          case TYPE_SERVICE.dedicatedServer:
+            // Update dedicated server
+            if (
+              !this.data.eligibleServices?.dedicatedServer?.length !==
+              result.dedicatedServer.length
+            ) {
+              this.data.eligibleServices.dedicatedServer =
+                result.dedicatedServer;
+
+              // Update dedicated server data
+              this.updateDedicatedServerServiceInfo();
+            }
+            break;
+          case TYPE_SERVICE.dedicatedServerInterface:
+            // Update dedicated server interface
+            if (
+              !this.data.eligibleServices?.dedicatedServerInterface?.length !==
+              result.dedicatedServerInterface.length
+            ) {
+              this.data.eligibleServices.dedicatedServerInterface =
+                result.dedicatedServerInterface;
+
+              // Update dedicated server interface
+              this.updateDedicatedServerInterfaceServiceInfo();
+            }
+            break;
+          case TYPE_SERVICE.ip:
+            // Update ip
+            if (!this.data.eligibleServices?.ip?.length !== result.ip.length) {
+              this.data.eligibleServices.ip = result.ip;
+
+              // Update IP service
+              this.updateIpServiceInfo();
+            }
+            break;
+          case TYPE_SERVICE.ipv6:
+            // Update ipv6
+            if (
+              !this.data.eligibleServices?.ipv6?.length !== result.ipv6.length
+            ) {
+              this.data.eligibleServices.ipv6 = result.ipv6;
+
+              // Update IPv6
+              this.updateIpv6ServiceInfo();
+            }
+            break;
+          case TYPE_SERVICE.ipLoadbalancing:
+            // Update load balancing
+            if (
+              !this.data.eligibleServices?.ipLoadbalancing?.length !==
+              result.ipLoadbalancing.length
+            ) {
+              this.data.eligibleServices.ipLoadbalancing =
+                result.ipLoadbalancing;
+
+              // Update IP Loadbalancing data
+              this.updateIpLoadbalancingServiceInfo();
+            }
+            break;
+          case TYPE_SERVICE.legacyVrack:
+            // Update legacy vRack
+            if (
+              !this.data.eligibleServices?.legacyVrack?.length !==
+              result.legacyVrack.length
+            ) {
+              this.data.eligibleServices.legacyVrack = result.legacyVrack;
+
+              // Update legacy vRack
+              this.updateLegacyVrackServiceInfo();
+            }
+            break;
+          default:
+            this.updateOvhCloudConnect();
+            break;
+        }
+      }
+    });
+  }
+
+  updateIpAndIpv6() {
+    if (this.data.eligibleServices?.ip?.length > 0) {
+      const ipv6List = this.data.eligibleServices.ip.filter((ip) =>
+        this.constructor.isIPv6(ip),
+      );
+      ipv6List.forEach((ipv6) => {
+        this.loadIpRegion(ipv6.id);
+      });
```

Comment:
filter function is useless here

```suggestion
      this.data.eligibleServices.ip.forEach((ip) => {
         if(this.constructor.isIPv6(ip)){
              this.loadIpRegion(ip.id);
         }
      });
```